### DISCUSSION TITLE
Support customizing the hovering text for DatePicker component

### DISCUSSION THREAD

**Discussion by AllForNothing** [vmware-clarity/ng-clarity#444] [General]
Currently,  it always shows "Choose date" when hovering on a DatePicker, this is not that friendly for an i18n project:
<img width="886" alt="image" src="https://user-images.githubusercontent.com/30999793/210714081-4c9364be-41e4-4ba2-8a4b-5f18389a0095.png">

If customization is supported, the corresponding text can be displayed according to the current language

**Comment by kevinbuhmann** [comment]
It is customizable. See https://clarity.design/documentation/internationalization. The property name for this string is `datepickerToggleChooseDateLabel`. You will also want to handle `datepickerToggleChangeDateLabel` for when a date is selected. See https://github.com/vmware-clarity/ng-clarity/blob/main/projects/angular/src/utils/i18n/common-strings.default.ts for all of the default strings.

**Reply by AllForNothing** [reply]
Thanks for the reply. 
 It works. I should have read the docs carefully！

**Comment by AllForNothing** [comment]
@kevinbuhmann The is a bug of the Datepicker component, https://github.com/vmware-clarity/ng-clarity/issues/450
